magento/magento2
# #3993; Fix same level categories (position)re-index on category move.

### Description (*)
When category is moved, the list of categories prepared for re-indexing now includes the sibling categories.

### Fixed Issues (if relevant)
1. magento/magento2#3993: Category Sort Order is not saved in Flat Table

### Manual testing scenarios (*)
1. Enable flat for categories.
2. Add some categories of the same level to any parent category.
3. Sort them in any order.
4. Check the corresponding flat table and see that position indices are calculated correctly.

### Questions or comments
--

### Contribution checklist (*)
